docs/source/quickstart.rst

```diff
@@ -288,7 +288,7 @@ server. Sometimes this will work fine. But it takes a little while for
 the server to start up and be ready to accept connections – so other
 times, randomly, our connection attempt will happen too quickly, and
 error out. After all – ``nursery.start_soon`` only promises that the
-task will be started *soon*, not that it's actually happened. So this
+task will be started *soon*, not that it has actually happened. So this
```

Comment:
"it's" as a contraction for "it has" is valid and common

Reply:
Yes it is common, but the contracion's validity depends on the emphasis being on the "it" part. In this sentence however the emphasis is on the "has", thus the contraction isn't approriate.